OBCDataContainer.h

```diff
@@ -0,0 +1,116 @@
+/*
+ *  OBCDataContainer.h
+ *
+ *  Created on: Jun 10, 2020
+ *      Author: tom-h
+ */
+
+#ifndef OBCDATACONTAINER_H_
+#define OBCDATACONTAINER_H_
+
+#define OBC_DATACONTAINER_SIZE  59
+
+typedef enum Mode {ACTIVATIONMODE, DEPLOYMENTMODE, SAFEMODE, ADCSMODE, NOMINALMODE} Mode;
+
+// States of Delfi-PQ in different mode
+typedef enum ActivationState  {UNEXPIRED, EXPIRED} ActivationState;
+typedef enum DeployState  {NORMAL, FORCED, DELAYING, DEPLOYED} DeployState;
+typedef enum ADCSState {IDLE, DETUMBLE, DISABLED} ADCSState;
+
+// Can be used in every mode for power line V2, V3 and V4
+typedef enum PowerState {INITIALIZING, INITIALIZED, CYCLING, OFF} PowerState;
+
+// Health check results of subsystems (TODO)

```

Comment:
Isnt it way easier to just use booleans here instead of an enum that has 2 possible values. The only reason for this I can think of is if we want to add further enum states later in which this implementation is fine